integration-tests/sync-tests/sync-verifier.spec.ts

```diff
@@ -0,0 +1,94 @@
+import chai, { expect } from 'chai'
+import { Wallet, BigNumber, providers } from 'ethers'
+import { injectL2Context } from '@eth-optimism/core-utils'
+
+import { sleep, l2Provider } from '../test/shared/utils'
+import { OptimismEnv } from '../test/shared/env'
+import { DockerComposeNetwork } from '../test/shared/docker-compose'
+
+describe('Syncing a verifier', () => {
+  let env: OptimismEnv
+  let wallet: Wallet
+  let verifier: DockerComposeNetwork
+
+  const provider = injectL2Context(l2Provider)
+
+  /* Helper functions */
+
+  const waitForBatchSubmission = async (totalElementsBefore: BigNumber) => {
```

Comment:
In an ideal world, we do not spin up the batch submitter and submit the batches manually using JS. This would cut out the extra dependency of needing to have a whole other codepath and the complexity that comes with it, like this function

Reply:
Given that these are integration tests checking the relation between the sequencer's sync service, the dtl, the verifier and the batch submitter, I think it's better as is actually

Reply:
I'm fine with leaving as is, but I think the more controlled the test, the better. This isn't a batch-submitter test, this is a verifier test. It should test the bare minimal stuff to make sure the verifier works.